three_regex.js

```diff
@@ -0,0 +1,18 @@
+// Напишіть regex, котрий знайде cat та can, але не знайде car
+var text = 'cat car can';
+var regex = /ca[t|n]/g;
```

Comment:
Instead of using | in square brackets, it is better to use a character set to match one of the characters. It is also necessary to take into account that can can be found in can or can't, so it is worth adding restrictions on spaces or the end of the line.

var regex = /\bca[t|n]\b/g;

Reply:
In general you are right and if want more precise search covering more cases and not only exampled - we need to use \b.
I'll update code.